packages/api-bindings/src/mocks.ts

```diff
@@ -1,2 +1,3 @@
 export * from './graphql/__helpers__';
 export * from './apollo/__helpers__/mocks';
+export { createProfileTypePolicy } from './apollo/createProfileTypePolicy';
```

Comment:
We need to access this in `FollowProfilesResponder.spec.ts` as part of the Apollo client setup.

Although not strictly a mock, we shouldn't be exporting as part of the main package

Reply:
We can avoid this by using the `createMockApolloCache` helper. See my other comment.